src/app/libp2p_helper/src/libp2p_helper/bitswap_msg.go

```diff
@@ -52,14 +52,14 @@ func extractRootBlockList(l ipc.RootBlockId_List) ([]root, error) {
 	return ids, nil
 }
 
-func (m DeleteResourcePush) handle(app *app) {
-	idsM, err := DeleteResourcePushT(m).Ids()
+func (m RemoveResourcePush) handle(app *app) {
```

Comment:
Is the _delete_ to _remove_ name change just for the sake of better names or there is a more explicit reason behind int?

Reply:
it looks like a name change for the sake of a better name; i haven't found a collision with the previous name